tensorflow_addons/conftest.py

```diff
@@ -0,0 +1,16 @@
+import tensorflow as tf
```

Comment:
Any reason this wouldn't fit under:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/utils/test_utils.py ?

Reply:
We can do it, but then we would have an unused import in conftest.py (fixtures need to be in a conftest.py to be refered with strings in tests.) I don't have a strong preference here so I'll move it to test_utils.py

Reply:
Done